CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Version master (UNRELEASED)
 
 - Adds `get_disk_usage` utility function to calculate disk usage for a directory.
 - Centralises `fs` package dependency
+- Changes `workflow-submission` queue as a priority queue and allows to set the priority number on workflow submission.
```

Comment:
reStructuredText uses double quotes for inline literals. I see that is also wrong in many other places in this file. We can take the opportunity to fix them.